newrelic/resource_newrelic_nrql_alert_condition_test.go

```diff
@@ -476,7 +476,7 @@ resource "newrelic_nrql_alert_condition" "foo" {
 		time_function = "any"
 	}
 
-	violation_time_limit_seconds = 3600
+	violation_time_limit_seconds = 86400
```

Comment:
Since this is the deprecated config here, do we still want to change this?

Reply:
No, not necessarily, but it does test the bug scenario.